### PR TITLE
Better wordings in the new user modal

### PR DESCRIPTION
**Pull request by Jerome-Herbinet**
I've made some wording improvements ; those are more clear, IMO.
Should do what you see below. Tests from reviewers required ; thanks by advance to them.

## Before 

# After 

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by Pytal — COMMENTED** [review]
Good idea though seems a bit long, will have to see what designers think :)

File: apps/settings/src/components/Users/NewUserModal.vue [hunk 1]
```diff
@@ -100,7 +100,7 @@
 				class="modal__item">
 				<label class="modal__label"
 					for="new-user-sub-admin">
-					{{ t('settings', 'Administered groups') }}
+					{{ t('settings', 'Manages user(s) in the following group(s)') }}
```
Comment (susnux) [inline apps/settings/src/components/Users/NewUserModal.vue:103]: I think the current wording is
```suggestion
					{{ t('settings', 'Manages accounts in the following group(s)') }}
```

File: apps/settings/src/components/Users/NewUserModal.vue [hunk 2]
```diff
@@ -77,7 +77,7 @@
 			<div class="modal__item">
 				<label class="modal__label"
 					for="new-user-groups">
-					{{ !settings.isAdmin ? t('settings', 'Groups (required)') : t('settings', 'Groups') }}
+					{{ !settings.isAdmin ? t('settings', 'Groups (required)') : t('settings', 'Belongs to the following group(s)') }}
```
Comment (jancborchardt) [inline apps/settings/src/components/Users/NewUserModal.vue:80]:
```suggestion
					{{ !settings.isAdmin ? t('settings', 'Groups (required)') : t('settings', 'Member of the following groups') }}
```

**Review by jancborchardt — CHANGES_REQUESTED** [review]
Added 2 suggestions, simplifying the language.

Also, we shouldn’t ever use the "(s)" singular/plural writing like "group(s)". Either we write only the plural (as in my suggestion), or we programmatically check:
- If nothing or multiple are put in, use plural
- If only 1 is put in, use singular

**Review by jancborchardt — APPROVED** [review]
Sounds good to me! :)

**Comment by jancborchardt** [comment]
@Pytal @susnux could you check again? :)

File: apps/settings/src/components/Users/NewUserModal.vue [hunk 3]
```diff
@@ -77,7 +77,7 @@
 			<div class="modal__item">
 				<label class="modal__label"
 					for="new-user-groups">
-					{{ !settings.isAdmin ? t('settings', 'Groups (required)') : t('settings', 'Groups') }}
+					{{ !settings.isAdmin ? t('settings', 'Groups (required)') : t('settings', 'Member of the following groups') }}
```
Comment (Pytal) [inline apps/settings/src/components/Users/NewUserModal.vue:80]:
```suggestion
					{{ !settings.isAdmin ? t('settings', 'Member of the following groups (required)') : t('settings', 'Member of the following groups') }}
```

**Comment by Pytal** [comment]
Looks good! Can you squash the commits @Jerome-Herbinet?

**Comment by susnux** [comment]
Squashed and rebased

**Review by susnux — APPROVED** [review]

**Review by Pytal — APPROVED** [review]
image
💃💃

**Pull request merged**